getting-started/plugin-features/README.md

```diff
@@ -3,101 +3,96 @@
 The following features are supported by the plugin, as listed below.
 
 - [Running Ballerina Programs](#running-ballerina-programs)
-- [Debugging Ballerina Programs](#debugging-ballerina-programs) 
-- [Diagram View](#diagram-view)
 - [Import modules On-the-fly](#import-modules-on-the-fly)
-- [Auto Import Unambiguous Modules](#auto-import-unambiguous-modules)
 - [Find Usage](#find-usage)
 - [Go to Definition](#go-to-definition)
 - [Formatting](#formatting)
-- [Parameter Information](#parameter-information)
+- [Signature help](#signature-help)
 - [Quick Documentation](#quick-documentation)
-- [Object/Record Fields suggestions](#objectrecord-fields-suggestions)
-- [Annotation fields suggestions](#annotation-fields-suggestions)
+- [Code completions](#code-completions)
+- [Code snippets](#code-snippets)
```

Comment:
Lets use the term `Live templates` as thats the common term among IntelliJ community
```suggestion
- [Live templates](#live-templates)
```